basics/getting-started/public-cloud-examples/README.md

```diff
@@ -1,12 +1,12 @@
 ---
 description: >-
-  This page contains multiple quick start guides for deploying Pinot to a public
-  cloud provider.
+  This page links to multiple quick start guides for deploying Pinot to different public
+  cloud providers.
 ---
 
 # Running on public clouds
 
-The following quick start guides will show you how to run an Apache Pinot cluster using Kubernetes on different public cloud providers.
+These quick start guides show you how to run an Apache Pinot cluster using Kubernetes on different public cloud providers.
```

Comment:
lets do global search for quick start, and Quick start, and Quickstart (other than at the beginning of sentence), and
globally replace with "quickstart" across the docs, per Google dev style